### PR TITLE
Add Remix wizard instructions

### PR DESCRIPTION
**Pull request by onurtemizkan**
Fixes: #7727
Continues: https://github.com/getsentry/sentry-docs/pull/7728

Can't push commits to the original PR, so opened a new one.

### CODE REVIEW

File: src/platform-includes/getting-started-install/javascript.remix.mdx
Comment (AbhiPrasad) [inline src/platform-includes/getting-started-install/javascript.remix.mdx:18]: We can remove this for Remix users.

```suggestion
```

File: src/platforms/javascript/guides/remix/manual-setup.mdx
Comment (lizokm) [inline src/platforms/javascript/guides/remix/manual-setup.mdx:9]:
```suggestion
## Installation

Get started by installing the below:
```

File: src/platforms/javascript/guides/remix/manual-setup.mdx
Comment (lizokm) [inline src/platforms/javascript/guides/remix/manual-setup.mdx:27]:
```suggestion
Create two files in the root directory of your project, `entry.client.tsx` and `entry.server.tsx` (if they don't already exist). Add your initialization code in these files for the client-side and server-side SDK, respectively.
```

File: src/platforms/javascript/guides/remix/manual-setup.mdx
Comment (lizokm) [inline src/platforms/javascript/guides/remix/manual-setup.mdx:142]:
```suggestion
If you use a custom Express server in your Remix application, you should wrap your [`createRequestHandler` function](https://remix.run/docs/en/v1/other-api/adapter#createrequesthandler) manually with `wrapExpressCreateRequestHandler`. This isn't necessary if you're using the built-in Remix App Server.
```

File: src/platforms/javascript/guides/remix/manual-setup.mdx
Comment (lizokm) [inline src/platforms/javascript/guides/remix/manual-setup.mdx:163]:
```suggestion
## Remix v2 Features
```

File: src/platforms/javascript/guides/remix/manual-setup.mdx
Comment (lizokm) [inline src/platforms/javascript/guides/remix/manual-setup.mdx:171]:
```suggestion
To capture errors from [v2 ErrorBoundary](https://remix.run/docs/en/main/route/error-boundary-v2), you should define your own `ErrorBoundary` in `root.tsx` and use `Sentry.captureRemixErrorBoundaryError` inside of it. You can also create route-specific error capturing behavior by defining `ErrorBoundary` in your route components. The `ErrorBoundary` you define in `root.tsx` will be used as a fallback for all routes.
```

File: src/platforms/javascript/guides/remix/manual-setup.mdx
Comment (lizokm) [inline src/platforms/javascript/guides/remix/manual-setup.mdx:187]:
```suggestion
Sentry won't be able to capture your server-side errors automatically if you're using the`v2_errorBoundary` future flag. To work around this, define a [`handleError`](https://remix.run/docs/en/main/file-conventions/entry.server#handleerror) function in your server entry point. Then use `Sentry.captureRemixServerError` to capture errors in your server-side code.
```

File: src/platforms/javascript/guides/remix/manual-setup.mdx
Comment (lizokm) [inline src/platforms/javascript/guides/remix/manual-setup.mdx:203]:
```suggestion
After you've completed this setup, the SDK will automatically capture unhandled errors and promise rejections, and monitor performance in the client. You can also [manually capture errors](/platforms/javascript/guides/remix/usage).
```

File: src/platforms/javascript/guides/remix/manual-setup.mdx
Comment (lizokm) [inline src/platforms/javascript/guides/remix/manual-setup.mdx:207]:
```suggestion
You can refer to [Remix Docs](https://remix.run/docs/en/v1/guides/envvars#browser-environment-variables) to learn how to use your Sentry DSN with environment variables.
```

**Review by lizokm — APPROVED** [review]
Made some suggestions, but otherwise this looks great!

File: src/platforms/javascript/guides/remix/manual-setup.mdx
Comment (Lms24) [inline src/platforms/javascript/guides/remix/manual-setup.mdx:140]: l: I'd prefer this section and the v2 support section to switch places unless there's a concrete reason to have custom express server instructions come first(?) My thinking is that showing that we support v2 (future flags) is more important than custom servers.

File: src/platforms/javascript/guides/remix/manual-setup.mdx
Comment (Lms24) [reply src/platforms/javascript/guides/remix/manual-setup.mdx:9]: l: Would just suggest a slight change here:
```suggestion
## Installation

Get started by installing the Sentry Remix SDK:
```

**Review by Lms24 — APPROVED** [review]
LGTM once the suggestions are applied, thanks Onur!

**Review by Lms24 — CHANGES_REQUESTED** [review]
Only requesting changes so we don't merge this until https://github.com/getsentry/sentry-wizard/issues/433 is resolved. Let's only publish this once the wizard is fully ready.

File: src/platform-includes/getting-started-install/javascript.remix.mdx
Comment (Lms24) [inline src/platform-includes/getting-started-install/javascript.remix.mdx:11]:
```suggestion
- create `.sentryclirc` with an auth token to upload source maps (this file is automatically added to `.gitignore`).
- adjust your `build` script in `package.json` to automatically upload source maps to Sentry.
```

**Review by Lms24 — APPROVED** [review]
We released the wizard with the org/project fix yesterday so this is ready to go. Just one small suggestion but otherwise ready to 🚢

File: src/platforms/javascript/guides/remix/manual-setup.mdx
Comment (shanamatthews) [inline src/platforms/javascript/guides/remix/manual-setup.mdx:84]: Looks like these links have moved

```suggestion
Initialize Sentry in your entry point for the server to capture exceptions and get performance metrics for your [`action`](https://remix.run/docs/en/1.19.3/route/action) and [`loader`](https://remix.run/docs/en/1.19.3/route/loader) functions. You can also initialize Sentry's database integrations, such as <Link to="/platforms/node/performance/database/opt-in/#prisma-orm-integration">Prisma</Link>, to get spans for your database calls.
```

**Review by shanamatthews — APPROVED** [review]

File: src/platforms/javascript/guides/remix/manual-setup.mdx
Comment (onurtemizkan) [inline src/platforms/javascript/guides/remix/manual-setup.mdx:84]: @shanamatthews - v2.0.0 is published so we can use `main` here instead. 

```suggestion
Initialize Sentry in your entry point for the server to capture exceptions and get performance metrics for your [`action`](https://remix.run/docs/en/main/route/action) and [`loader`](https://remix.run/docs/en/main/route/loader) functions. You can also initialize Sentry's database integrations, such as <Link to="/platforms/node/performance/database/opt-in/#prisma-orm-integration">Prisma</Link>, to get spans for your database calls.
```

**Pull request merged**
